### ISSUE TITLE
python-wand font generation sometimes times out

### ISSUE THREAD

**Issue by d3zd3z** [OP-TEE/optee_os#868]
Occasionally, when I build optee_os, I see an error like this:

```make
| python2.7: time limit exceeded `Operation canceled' @ fatal/cache.c/GetImagePixelCache/1880.
| python2.7: time limit exceeded `Operation canceled' @ fatal/cache.c/GetImagePixelCache/1880.
| make: *** [mk/subdir.mk:152: out/arm-plat-vexpress/ta_arm32-lib/libutee/tui/default_bold.c] Error 1
```

ImageMagick has a global timeout value, but there doesn't appear to be a way of setting it within the python-wand library.

**Comment by d3zd3z** [comment]
ImageMagick documentation suggests that MAGICK_TIME_LIMIT can be set in the environment to override this value.  There is some other suggestions that the default time limit may be broken in some versions, since the default should be much longer than the errors I'm seeing.

**Comment by jbech-linaro** [comment]
Is this still an issue @d3zd3z ? From what I can recall I've never seen this myself?

**Comment by jbech-linaro** [comment]
TUI stuff closed in #1844 .

**Comment by jbech-optee-adm** [comment]
We are closing this issue/question, why?
 - We believe the question has been answered.
 - Quite often when we have answered the questions, the issuer forget to close the issue which results in many open issues for us which makes it hard to keep track of the ones actually waiting for an answer.
 - In some cases we ask follow-up questions, but do not hear back from the one creating the issue.

If you however feel that you have additional questions or still thinks this is an issue, then please feel free to re-open the issue again. When closing issues we will only show this entire message once.

// OP-TEE admins
